platform/ios/src/MGLTelemetryConfig.h

```diff
@@ -0,0 +1,18 @@
+#import <Foundation/Foundation.h>
+#import <CoreLocation/CoreLocation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface MGLTelemetryConfig : NSObject
+
+@property (nonatomic) CLLocationDistance MGLLocationManagerHibernationRadius;
```

Comment:
nit: Removing the "MGL" on this property might make it little easier to use at the call site (i.e. `.locationManagerHibernationRadius`). No need to change it here but something to think about when this is ported to the events lib.